maxim/tests/openai/agents/cookbook_test/cookbook_agents.py

```
@@ -0,0 +1,117 @@
+
+
+def upload_file(file_path: str, vector_store_id: str):
+    file_name = os.path.basename(file_path)
+    try:
+        file_response = client.files.create(
+            file=open(file_path, "rb"), purpose="assistants"
+        )
+        attach_response = client.vector_stores.files.create(
+            vector_store_id=vector_store_id, file_id=file_response.id
+        )
+        return {"file": file_name, "status": "success"}
```

Ruff (0.14.4)

Consider moving this statement to an `else` block

(TRY300)